pkg/operator/ceph/cluster/nodedaemon/ceph-exporter.go

```diff
@@ -0,0 +1,239 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodedaemon
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strconv"
+
+	"github.com/pkg/errors"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/rook/rook/pkg/operator/ceph/config"
+	"github.com/rook/rook/pkg/operator/ceph/controller"
+
+	cephver "github.com/rook/rook/pkg/operator/ceph/version"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	monitoringPath               = "/etc/ceph-monitoring/"
+	serviceMonitorFile           = "ceph-exporter-service-monitor.yaml"
+	cephConfFile                 = "/var/lib/rook/rook-ceph/rook-ceph.config"
+	configDir                    = "/var/lib/rook/rook-ceph"
```

Comment:
Well actually this path need to be fixed for exporter, because the config it'll use has 
admin keyring(`/var/lib/rook/rook-ceph/admin.keyring`) path hard coded in it which exporter daemon will use for it's creation.